com.discordapp.Discord.json

```diff
@@ -17,9 +17,6 @@
         "--share=network",
         "--device=all",
         "--talk-name=org.freedesktop.Notifications",
-        "--filesystem=xdg-videos:ro",
-        "--filesystem=xdg-pictures:ro",
```

Comment:
We may want to keep these so drag-n-drop from them works.

Reply:
Hm, forgot about drag-n-drop, I'll just keep this PR open as a draft if that ever gets resolved

Reply:
does it work in Chromium? I mean is it only Electron issue, or Chromium as well?

Reply:
No, there is an [open issue](https://bugs.chromium.org/p/chromium/issues/detail?id=1424845) on the Chromium bug tracker

Reply:
xdg-download write access could be removed right now tho, as it's not necessary for drag-n-drop